Userland/Libraries/LibGfx/Bitmap.cpp

```diff
@@ -37,6 +37,7 @@ size_t Bitmap::minimum_pitch(size_t width, BitmapFormat format)
     switch (determine_storage_format(format)) {
```

Comment:
Overall: could you add a small test case so we can prevent regressions?

Reply:
Done

Reply:
Could you also test the actual color data?

Reply:
It looks like I do now